apps/server/src/modules/video-conference/bbb/bbb.service.ts

```diff
@@ -111,15 +114,19 @@ export class BBBService {
 					BBBResponse<BBBMeetingInfoResponse> | BBBResponse<BBBBaseResponse>
 				>(resp.data);
 				if (bbbResp.response.returncode !== BBBStatus.SUCCESS) {
-					throw new InternalServerErrorException(bbbResp.response.messageKey, bbbResp.response.message);
+					throw new InternalServerErrorException(`${bbbResp.response.messageKey}: ${bbbResp.response.message}`);
 				}
 				return bbbResp as BBBResponse<BBBMeetingInfoResponse>;
 			})
 			.catch((error) => {
-				throw new InternalServerErrorException(error);
+				throw new InternalServerErrorException(
+					null,
+					ErrorUtils.createHttpExceptionOptions(error, 'BBBService:getMeetingInfo')
+				);
 			});
 	}
 
+	// should be private
```

Comment:
```suggestion
	// TODO: should be private
```
So one can find it easier.

Reply:
TODO: 
dürfen nicht mehr gemerged werden.